src/feature/flow_chart_panel/views/AppGalleryModal.tsx

```diff
@@ -43,7 +58,7 @@ export const AppGalleryModal = ({
       const filtered = raw.filter(
         (obj) => obj["type"] === "dir" && !ignoreDir.includes(obj["name"])
       );
-      setSelect(
+      setSelectFields(
         filtered.map((obj) => (
           <SelectItem key={obj.sha} value={obj.url}>
             {obj.name}
```

Comment:
Instead of storing an array of components as state, I think it would be better to just store the data then map it to the `SelectItems` in the `SelectContent`